Explorer/Assets/DCL/Multiplayer/Emotes/MultiplayerEmotesMessageBus.cs

```diff
@@ -74,19 +74,20 @@ public void Send(URN emote, bool loopCyclePassed)
         public void OnPlayerRemoved(string walletId) =>
             messageDeduplication.RemoveWallet(walletId);
 
-        private void SendTo(URN emoteId, uint timestamp, IMessagePipe messagePipe)
+        private void SendTo(URN emoteId, uint incrementalId, IMessagePipe messagePipe)
         {
             MessageWrap<PlayerEmote> emote = messagePipe.NewMessage<PlayerEmote>();
 
             emote.Payload.Urn = emoteId;
-            emote.Payload.IncrementalId = timestamp;
+            emote.Payload.IncrementalId = incrementalId;
+            emote.Payload.Timestamp = UnityEngine.Time.unscaledTime;
```

Comment:
Do we really need `IncreamentalId` and `Timestamp`? Can `Timestamp` fulfill responsibilities that currently fulfilled by `IncreamentalId`? Probably, it is just ordering logic and `Timestamp `can easily be used there. This way we don't increase the bandwidth with the proto change. From another hand, it will be not backward compatible, so we would need to inform proto squad

Reply:
Done